README.md

```diff
@@ -817,9 +818,28 @@ Then pass the generated `custom_genesis_params` `dict` to the backend's `__init_
 >>> t = EthereumTester(backend=pyevm_backend)
 ```
 
-Overriding genesis state is similar to overriding genesis parameters but requires the consideration of test accounts.
-To override the genesis state of accounts, pass a `state_overrides` `dict` to `PyEVMBackend._generate_genesis_state`,
-and optionally, the number of accounts to create.  
+To override the genesis state of accounts, pass a `state_overrides` `dict` to `PyEVMBackend._generate_genesis_state`.
```

Comment:
I would really prefer to call a public method.

Can we make the same method except public that passthroughs to the protected one? That way, anyone who followed this readme won't have broken stuff and new users, like myself, can use the public one.

It does not have to happen on this PR but it would be great and I can take care of it

Reply:
https://github.com/ethereum/eth-tester/pull/210

^ If this merges first, I will adjust this PR accordingly.

Reply:
Yep, that sounds good to me!